programs/mango-v4/src/instructions/serum3_liq_force_cancel_orders.rs

```diff
@@ -63,7 +63,7 @@ pub fn serum3_liq_force_cancel_orders(
         {
             let result = account.check_liquidatable(&health_cache);
             if account.fixed.is_operational() {
-                if !result? {
+                if !result? && !serum_market.is_force_close() {
```

Comment:
This won't work: check_liquidatable() is tricky:
- returns Ok(true) when liquidatable
- returns Ok(false) if the account recovered from being_liquidated
- returns Err(HealthMustBeNegative) if account isn't liquidatable and wasn't liquidatable
- might return Err(something_else)
I think we should change it to just return a nice enum, then this code becomes less confusing too.